### ISSUE TITLE
Option to dim / disable display brightness

### ISSUE THREAD

**Issue by japaweiss** [doidotech/TBM_Node#2]
As a user I want to be able to dim / disable the display brightness because in some situation it is too bright for the room

**Comment by AronGahagan** [comment]
Agree. Would also like to turn it off at night, say between 11 PM and 5 AM.
